### PR TITLE
adds berty's profile

### PR DESCRIPTION
**Pull request by bertyruan**

### CODE REVIEW

File: Submissions/bertyruan.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Berty Ruan",
+  "img": "https://avatars.githubusercontent.com/u/5183841?v=4",
+  "email": "berty.ruan@gmail.com",
+  "links": {
+    "website": "bertydev.com",
```
Comment (ProsperousHeart) [inline Submissions/bertyruan.json:6]: please provide the full site, including the http or https

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
1 minor change requested

**Comment by bertyruan** [comment]
Hi @ProsperousHeart thanks for the comment. I've made the requested changes. Could you review my PR for one of ZtM's open source projects? I've been pinging the admins and project owners but no one's paying attention hahaha https://github.com/zero-to-mastery/CSS_Grid_LearnGame/pull/59

**Review by ProsperousHeart — APPROVED** [review]
Approved!

**Pull request merged**